database/test_database.py

```diff
@@ -0,0 +1,103 @@
+from unittest import TestCase
+
+from database.image import *
+from database.project import *
+
+
+# Before running make sure no .db files are present in execution directory
```

Comment:
Please, to run the unit tests use an in memory sqlite database. https://www.sqlite.org/inmemorydb.html